### DISCUSSION TITLE
Is there a way to disable the Trigger Click on Dualsense Controllers?

### DISCUSSION THREAD

**Discussion by suavebeardson** [darthcloud/BlueRetro#1007] [Q&A] [answered]
Don't get me wrong, the trigger click is a great feature
I speedrun Metal Gear Solid The Twin Snakes on Emulator and got so used to my controller setup on there that I'm not a fan of the trigger click for this particular game.  Setting this up to speedrun on console I originally just used the adapter out of the box without update and just recently updated the firmware to 24.04

**Comment by darthcloud** [comment, marked as answer]
The config of that is tied to the rumble feature. Disable BlueRetro rumble in the output config.

**Reply by suavebeardson** [reply]
awesome thank you for the response
